src/pages/resultsView/download/DownloadTab.tsx

```diff
@@ -373,14 +378,57 @@ export default class DownloadTab extends React.Component<IDownloadTabProps, {}>
         );
     }
 
+    private copyDownloadQueryControlsRow(title:string,
+                                    handleDownload: () => string,
+                                    filename: string,
+                                    handleQuery: () => void,
+                                    handleVisualStudy:() => void)
+    {
+        return (
+            <tr>
+                <td>{title}</td>
+                <td>
+                    <SimpleCopyDownloadControls
+                        controlsStyle='QUERY'
+                        downloadData={handleDownload}
+                        downloadFilename={filename}
+                        handleQuery={handleQuery}
+                        handleVisualStudy={handleVisualStudy}
+                    />
+                </td>
+            </tr>
+        );
+    }
+
     readonly alteredSamplesDownloadControls = remoteData({
-        await:()=>[this.alteredSamplesText],
+        await:()=>[this.alteredCaseAlterationData],
+        invoke:()=>{
+            const alteredSampleIds = _.map(this.alteredCaseAlterationData.result!, caseAlteration => `${caseAlteration.studyId}:${caseAlteration.sampleId}`);
+            const handleDownload = () => alteredSampleIds.join("\n");
+            const handleQuery = () => this.handleQueryButtonClick(alteredSampleIds.join("\n"));
```

Comment:
would be nice if did not serialize these to strong until we actually need to.   like, the store property that holds these should be an array, not a string

Reply:
same with alteredSampleIds.   Someone reading code would think this is an array